sycl/test/basic_tests/image.cpp

```diff
@@ -24,6 +24,9 @@ int main() {
   const sycl::image_channel_order ChanOrder = sycl::image_channel_order::rgba;
   const sycl::image_channel_type ChanType = sycl::image_channel_type::fp32;
 
+  constexpr auto SYCLRead = sycl::access::mode::read;
```

Comment:
Are these supposed to have different namespace listings?

Reply:
No, will fix.

Reply:
Fixed.